### PR TITLE
Moving to ERC721

### PR DESCRIPTION
**Pull request by sullof**
Removing the enumerability of the erc721

### CODE REVIEW

File: scripts/deploy-passes.js [hunk 1]
```diff
@@ -48,10 +48,8 @@ async function main() {
   assert.isTrue(validator.length === 42)
   // assert.isTrue(operator.length === 42)
 
-  const maxTokenId = process.env.MAX_TOKEN_ID || 888
```
Comment (superpower007) [inline scripts/deploy-passes.js:51]: why use ||?

File: scripts/deploy-passes.js [hunk 2]
```diff
@@ -48,10 +48,8 @@ async function main() {
...
-
   const SynCityPasses = await ethers.getContractFactory("SynCityPasses")
-  const nft = await SynCityPasses.deploy(maxTokenId, validator)
+  const nft = await SynCityPasses.deploy(validator)
```
Comment (superpower007) [inline scripts/deploy-passes.js:52]: why deploy changes without constructor changing.  was it wrong ?

File: scripts/deploy-passes.js (hunk 1 above)
Comment (sullof) [reply scripts/deploy-passes.js:51]: Because during testing I deploy much less token

Comment (sullof) [reply scripts/deploy-passes.js:51]: In some case you want to force a different size. But it is obsolete, now it is fixed
I will remove it

File: scripts/deploy-passes.js (hunk 2 above)
Comment (sullof) [reply scripts/deploy-passes.js:52]: We made it fixed

**Pull request merged**